app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -11,11 +11,11 @@ def description
   def connect(options = {})
     # raise "no credentials defined" if missing_credentials?(options[:auth_type])
 
-    username = options[:user] || authentication_userid(options[:auth_type])
-    password = options[:pass] || authentication_password(options[:auth_type])
-    host     = options[:host]
-
-    self.class.raw_connect(username, password, host)
+    username   = options[:user] || authentication_userid(options[:auth_type])
+    password   = options[:pass] || authentication_password(options[:auth_type])
+    host       = options[:host]
+    verify_ssl = options[:verify_ssl] ||
+    self.class.raw_connect(username, password, host, verify_ssl)
```

Comment:
Ha I just went to see why this was there, and realised the file isn't checked in with the extra `||`